flink-table/flink-sql-client/src/test/resources/sql/catalog_database.q

```diff
@@ -203,10 +203,12 @@ describe hivecatalog.`default`.param_types_table;
 !ok
 
 SET execution.type = batch;
+[WARNING] The specified key 'execution.type' is deprecated. Please use 'execution.runtime-mode' instead. Set the corresponding config option and the specified key together.
```

Comment:
Personally, I suggest to use new config keys. And add deprecated key tests in `set.q`.